web/themes/custom/votegov/templates/component/footer--secondary-section.html.twig

```diff
@@ -1,24 +1,31 @@
+{% set GSAlink = link('General Services Administration' | t, 'https://www.gsa.gov/') %}
+
 <div class="usa-footer__secondary-section">
   <div class="grid-container">
-    <div class="grid-row grid-gap">
-      <div class="usa-footer__logo grid-row mobile-lg:grid-col-6 mobile-lg:grid-gap-2">
-        <div class="mobile-lg:grid-col-auto">
-          {# Footer Secondary menu. #}
-          {{ 'Menu Placeholder' }}
-        </div>
-      </div>
-      <div class="usa-footer__contact-links mobile-lg:grid-col-6">
-        {# Social links menu. #}
-        <div class="usa-footer__social-links grid-row grid-gap-1">
-          <div class="grid-col-auto">
-            <a class="usa-social-link" href="javascript:void(0);"
-            ><img
-                class="usa-social-link__icon"
-                src="{{ uswds_img_path }}/usa-icons/twitter.svg"
-                alt="Twitter"
-              /></a>
+    <div class="usa-footer__logo">
+      <div class="partner-logo-menu">
```

Comment:
I think wrapper isn't needed after you move the social links out.
Remove `<div class="partner-logo-menu">`